packages/labs/gen-wrapper-vue/src/lib/wrapper-module-template-sfc.ts

```diff
@@ -111,6 +111,9 @@ const wrapperTemplate = (
 ) => {
   const {tagname, events, reactiveProperties} = declaration;
   return javascript`
+    <script lang="ts">
+      export * from '${wcPath}';
```

Comment:
Let's change this to export the type references that were imported.

Reply:
Fixed.